homeassistant/components/blebox/translations/bg.json

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
We should leave the existing files in place without changes.  lokalize will take care of updating after this merges

Reply:
Files restored, en.json changed only.